pom.xml

```diff
@@ -102,10 +103,9 @@
         <module>extensions/azure-extensions</module>
         <module>extensions/namespace-lookup</module>
         <module>extensions/kafka-extraction-namespace</module>
-
-        <!-- distribution packaging -->
```

Comment:
adding again, was a side effect of resolving conflicts during rebase against latest code.